_episodes/05-history.md

```diff
@@ -348,6 +348,30 @@ moving backward and forward in time becomes much easier.
 > 4. `$ git checkout <unique ID of last commit> data_cruncher.py`
 >
 > 5. Both 2 and 4
+>
+> > ## Solution
+> >
+> > Lets look at each command to see what will happen.
+> > 
+> > 1. `$ git checkout HEAD`
+> >
+> > When Jennifer runs this command, git will not change any of her files because this command does not specify which files she wants to checkout.
+> >
+> > 2. `$ git checkout HEAD data_cruncher.py`
+> >
+> > With this command, git will replace Jennifer's "broken" copy of data_cruncher.py with the previous commit's unmodified version.  She'll be given a "pre-break" file and have a chance to try editing it again.
```

Comment:
"broken" is jargon. You can either use "current" or omit this word at all.
"pre-break" is jargon as well. In fact, I don't think we need the second sentence altogether.